front/src/pods/student/student.component.tsx

```diff
@@ -11,25 +16,44 @@ interface Props {
 export const StudentComponent: React.FC<Props> = props => {
   const { room, log } = props;
 
+  const [autoScroll, setAutoScroll] = React.useState(false);
+
+  const textAreaRef = React.useRef(null);
+
+  React.useEffect(() => {
+    if (!autoScroll)
+      textAreaRef.current.scrollTop = textAreaRef.current.scrollHeight;
```

Comment:
Taking the update form the previous it  you don't need the not operator, and on the other hand let's apply some defensive programming, _textAreaRef_ will always hold a value, but image that somebody decides to hide the control temporary by some reason, rather avoid a bad crash here.

```tsx
if (isAutoScrollEnabled && textAreaRef.current) {
      textAreaRef.current.scrollTop = textAreaRef.current.scrollHeight;
}
```

On important thing always add the curly bracket to an if statement, even if it's one line

https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces